src/bun.js/bindings/headers-handwritten.h

```diff
@@ -138,14 +139,9 @@ const ZigStackFrameCode ZigStackFrameCodeWasm = 5;
 const ZigStackFrameCode ZigStackFrameCodeConstructor = 6;
 
 typedef struct ZigStackFramePosition {
-    int32_t source_offset;
-    int32_t line;
-    int32_t line_start;
-    int32_t line_stop;
-    int32_t column_start;
-    int32_t column_stop;
-    int32_t expression_start;
-    int32_t expression_stop;
+    WTF::OrdinalNumber line;
+    WTF::OrdinalNumber column;
+    int byte_position;
```

Comment:
Let's be more specific so there's no ambiguity:
```suggestion
    int32_t byte_position;
```